src/fragmentarium/ui/image-annotation/annotation-tool/FragmentAnnotation.tsx

```diff
@@ -236,12 +236,15 @@ function FragmentAnnotation({
         setToggled(null)
         setIsChangeExistingMode(false)
       } else if (geometry) {
-        const newAnnotation = new Annotation(geometry, {
-          ...data,
-          id: uuid4(),
-        })
-        setAnnotation({})
-        setAnnotations([...annotations, newAnnotation])
+        const minSize = Math.min(geometry.height, geometry.width)
+        if (minSize >= 0.3) {
```

Comment:
Extract method with a proper name to clarify what is going on. Avoid magic numbers.

Reply:
If you extract a method it also becomes testable.